ADB/Healtcheck.py

```diff
@@ -0,0 +1,44 @@
+import os
+import requests
+import toml
+
+
+def get_api_key():
```

Comment:
As I mentioned earlier (https://github.com/sustainableaviation/demandmap/pull/20#discussion_r1584561530), this is all fine for now. Before we release the code, you will need to add Numpy-style doctrings.